--- ../inc/common.php.original	2011-05-15 22:55:15.000000000 +0900
+++ ../inc/common.php	2011-05-15 22:55:52.000000000 +0900
@@ -828,8 +828,10 @@
             if(empty($data['tplfile'])){
                 $path = dirname(wikiFN($id));
                 $tpl = '';
-                if(@file_exists($path.'/_template.txt')){
-                    $data['tplfile'] = $path.'/_template.txt';
+                // modified by harukasan
+                // _template.txt -> template.txt
+                if(@file_exists($path.'/template.txt')){
+                    $data['tplfile'] = $path.'/template.txt';
                 }else{
                     // search upper namespaces for templates
                     $len = strlen(rtrim($conf['datadir'],'/'));
